source/_patterns/components/secondary-navigation/secondary-nav.hbs

```diff
@@ -1,6 +1,6 @@
   <ul class="pf-secondary-nav" role="tablist">
     <li class="pf-secondary-nav__item">
-      <a href="#" role="tab" class="pf-secondary-nav__link" aria-selected="true">Secondary Tab One</a>
+      <a href="#" role="tab" class="pf-secondary-nav__link is-active" aria-selected="true">Secondary Tab One</a>
```

Comment:
The more I think about this and read your comments, the more undecided I am. Without actually using something and testing it out, I can't say if I like it or now. And my psychic powers are failing me today.
![bitmoji](https://render.bitstrips.com/v2/cpanel/8810031-170621439_1-s4-v1.png?transparent=1&palette=1&width=123)